CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log for corePKCS11 Library
 
+## v3.5.0 (August 2022)
+* [#156](https://github.com/FreeRTOS/corePKCS11/pull/156) MISRA compliance updates
+* [#155](https://github.com/FreeRTOS/corePKCS11/pull/155) Update CBMC Starter kit
+* [#150](https://github.com/FreeRTOS/corePKCS11/pull/150) Add mbedtls 3.X support to core_pkcs11_mbedtls.c
```

Comment:
Is it backwards compatible with older versions ? If so please mention that as well.

Reply:
It should be according to the PR, added it to the message.